Makefile

```diff
@@ -5,6 +5,11 @@ all: build
 GOPATH ?= $(HOME)/go
 PATH := $(GOPATH)/bin:$(PATH)
 
+
+# Set the target version of macOS. This is an ENV that is consumed by go build (and downstream c pieces)
+# It's set here, and not in the GitHub Actions tooling, so that we notice build warnings.
+export MACOSX_DEPLOYMENT_TARGET = 11
```

Comment:
I can't decide if this is better in the Makefile, or the workflow. I opted to put it here, on the theory that our normal builds should follow as much of the same path.

_But_ it's a small lie -- changes to this don't rebuild cached files. So if you _really_ want it, you need to also run `go clean --cache`